server/game/cards/02_SHD/units/OmegaPartOfTheSquad.ts

```diff
@@ -27,7 +27,7 @@ export default class OmegaPartOfTheSquad extends NonLeaderUnitCard {
             targetZoneFilter: WildcardZoneName.AnyArena,
             ongoingEffect: AbilityHelper.ongoingEffects.ignoreAllAspectPenalties({
                 cardTypeFilter: WildcardCardType.NonLeaderUnit,
-                match: (card) => this.isFirstClonePlayedByControllerThisPhase(card),
+                match: (card) => card.hasSomeTrait(Trait.Clone) && this.isFirstClonePlayedByControllerThisPhase(card),
```

Comment:
maybe you should add this condition into isFirstClonePlayedByControllerThisPhase?

Reply:
+1

Reply:
moved condition